packages/karma-puppeteer-launcher/src/index.cjs

```diff
@@ -50,6 +51,10 @@ function puppeteerBrowser(baseBrowserDecorator, config) {
         // See http://crbug.com/715363
         // We use this flag to work-around this issue.
         '--disable-dev-shm-usage',
+        // Added to prevent disconnect timeouts: sets the maximum GPU memory to use for discardable caches.
+        '--force-gpu-mem-discardable-limit-mb',
+        // https://stackoverflow.com/questions/67501093/passthrough-is-not-supported-gl-is-disabled
+        '--disable-software-rasterizer',
```

Comment:
So this is just to get rid of some error message? Does the error crash the tests or just pollute the logs?

Reply:
just pollute the logs remotely, this seems to also maintain headless working when it is run locally.